documentation/modules/ROOT/pages/hcp-tech-breakdown.adoc

```diff
@@ -29,45 +31,53 @@ This operator is versioned in lockstep with a specific OCP version and is decoup
 
 The main responsibilities of this operator are:
 
-* Reading CAs from the hosted cluster to configure the kube controller manager CA bundle running in the hosted control plane.
+* Reading CAs from the hosted cluster(it is not clear what is hosted cluster here and how the read happens?) to configure the kube controller manager CA bundle running in the hosted control plane.
 * Reconciling resources that live on the hosted cluster such as `ClusterVersion`, `CatalogSources`, `OAuth`, etc.
 
 [#hosted-control-planes-networking]
 == Hosted Control Planes Networking
+(maybe just Networking in the Management cluster that hosts HCPs?)
 
-In Hosted Control Plane the control plane and the data plane are decoupled, this means that they may live in different infrastructures. These infrastructures may not be directly connected and as such, there are some networking differences between a regular OpenShift cluster and a hosted OCP cluster.
+In Hosted Control Plane the control plane and the data plane (what is really data plane?) are decoupled, this means that they may live in different infrastructures. These infrastructures may not be directly connected and as such, there are some networking differences between a regular OpenShift cluster and a hosted OCP cluster.
 
 The Hosted Control Plane and Data Plane use the https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication/#konnectivity-service[_Konnectivity_ service] to establish a tunnel for communications from the control plane to the data plane. This connection works as follows:
 
 1. The Konnectivity agent in the compute nodes, connects with the Konnectivity server running as part of the Hosted Control Plane.
-2. The Kubernetes API Server uses this tunnel to communicate with the compute nodes Kubelet.
+2. The Kubernetes API Server uses this tunnel to communicate with the compute nodes Kubelet (except logs the connection is from kubelet to api, not the other way).
 3. The compute nodes reach the Hosted Cluster API via an exposed service. Depending on the infrastructure where the Hosted Control Plane runs this service can be exposed via a load balancer, a node port, etc.
+(confused what is diff between Kubernetes API Server in previous bullet and Hosted Cluster API)
 4. The Konnecitivity server is used by the Hosted Control Plane to consume services deployed in the hosted cluster namespace such as OLM, OpenShift Aggregated API and OAuth.  
+( I was under the impression that are also part of the control as coexist in the managements cluster, what are exactly the components live on the control and on the compute nodes)
 
 Below image depicts these connections:
 
 image::hcp-dp-connection.png[hosted-control-plane-data-plane-connections]
 
+(not show clear, could expand this for two HostedClusters?, also what is KAS, Guest? is the kubelet speaks control traffic over the data plane?)
```

Comment:
two hosted will be the same, you will just have two hostedcluster boxes and two guest data plane boxes. KAS is Kubernetes API Server. Kubelet will talk thru the exposed service (LoadBalancer SVC or NodePort).